content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/manually.rst

```diff
@@ -0,0 +1,109 @@
+==================
+Manual reservation
+==================
+
+Unlike the *At Confirmation* reservation method, the *Manually* reservation method does **not**
+reserve products automatically.
+
+Instead, once a sales order (SO) is confirmed, product availability **must** be checked manually,
+and the required quantity **must** be reserved manually.
+
+.. seealso::
+   :doc:`About reservation methods <../reservation_methods>`
+
+Configuration
+=============
+
+To set the reservation method to *Manually*, navigate to :menuselection:`Inventory app -->
+Configuration --> Operations Types`.
+
+In the :guilabel:`General` tab, locate the :guilabel:`Reservation Method` field, and select
+:guilabel:`Manually`.
+
+.. image:: manually/manually-operations-type.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+Workflow
+========
+
+To see the *Manually* reservation method in action, create a new :abbr:`SO (sales order)` by
+navigating to :menuselection:`Sales app --> New`.
+
+Add a customer in the :guilabel:`Customer` field. Then, in the :guilabel:`Order Lines` tab, click
+:guilabel:`Add a product`, and select a product to add to the quotation from the drop-down menu.
+Finally, in the :guilabel:`Quantity` column, adjust the desired quantity of the product to sell.
+
+Once ready, click :guilabel:`Confirm` to confirm the sales order.
+
+Click the green :guilabel:`📈 (forecast graph)` icon on the product line to reveal the product's
+:guilabel:`Availability` tooltip. This tooltip reveals the reserved number of units for this order.
+Because the reservation method is set to *Manually*, the :guilabel:`Reserved` quantity reads `0
+Units`.
+
+However, below that quantity reads `Available in stock`. This is because the quantity is available,
+but must be manually reserved.
```

Comment:
Same with the other related PR - going to look into this one and will update accordingly

Reply:
Did some tests and found that the tooltip presents different info based on different factors. So I added this note in after this sentence:

.. note::
   If there is **not** sufficient quantity of stock for the product included in the |SO|, the
   :guilabel:`📈 (forecast graph)` icon is red, instead of green.

   Instead of revealing the reserved number of units for the order, the :guilabel:`Availability`
   tooltip reads :guilabel:`Reserved`, and reveals the available number of units (e.g., `0 Units`).

   Additionally, unless there is a set replenishment or a live receipt, it also reads :guilabel:`No
   future availability`, in red text.

Suggestion implemented ✅